src/cells/model.ts

```diff
@@ -429,6 +438,15 @@ class CodeCellModel extends BaseCellModel implements ICodeCellModel {
   }
 
   /**
+   * Clear the cell state.
+   */
+  clear(): void {
+    this.output.clear(false);
+    this.executionCount = null;
+    this.input.prompt = 'In [ ]:';
```

Comment:
Should it also have a say in what the prompt is for busy and given an execution count then?